src/components/UserSelector.tsx

```diff
@@ -1,34 +1,26 @@
-import React, { useContext, useEffect, useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import classNames from 'classnames';
-import { UserContext } from './UsersContext';
-import { User } from '../types/User';
+import { useAppDispatch, useAppSelector } from '../app/hooks';
+import * as usersSliceActions from '../features/users';
+import * as authorSliceActions from '../features/author';
```

Comment:
Ensure that the import path '../features/author' correctly points to the file where the `authorSliceActions` are defined. If the path is incorrect, it will cause runtime errors.